mmrotate/visualization/local_visualizer.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Optional
+
+import numpy as np
+import torch
+from mmdet.structures.mask import bitmap_to_polygon
+from mmdet.visualization import DetLocalVisualizer, get_palette
+from mmdet.visualization.palette import _get_adaptive_scales
```

Comment:
`from mmdet.visualization import _get_adaptive_scales`

mmdet.visualization.palette will be deleted.